pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -73,10 +73,16 @@ in stdenv.mkDerivation rec {
     "--with-udevdir=${placeholder "out"}/lib/udev"
     "--enable-health"
     "--enable-mesh"
-    "--enable-midi"
     "--enable-nfc"
     "--enable-sap"
     "--enable-sixaxis"
+    "--enable-midi"
+    "--enable-btpclient"
+    "--enable-experimental"
+    "--enable-deprecated"
```

Comment:
I'm not familiar with the upstream project, but asking for experimental and deprecated features in the nixpkgs package by default doesn't sound that good to me.

Reply:
>My motivation is to open unix pipe over the air using serial port profile and it seems currently there is no easy alternative tool (except --enable-deprecated).

Well that's... frustrating. (Is there an upstream issue?). What about `--enable-experimental`? Does it really belong here?

Reply:
`--enable-experimental` does not strictly belong here. It should be removed, please wait.

Reply:
Maybe you can add a comment about the need for `--enable-deprecated` in the code?

Reply:
also removed `--enable-admin` for making sure since it provides a plugin to the daemon.